documentation/maintenance-page.md

```diff
@@ -52,6 +52,7 @@ flowchart LR;
 - Check the domains and kubernetes resource names in `new_service/maintenance_page/manifests`
 - Only production and development environments are provided as examples. Copy and amend for new environments.
 - Create workflows for enable-maintenance and disable-maintenance similar to https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/.github/workflows/enable-maintenance.yml and https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/.github/workflows/disable-maintenance.yml
+        The enable-maintenance workflow above should a package with the correct permission and visibility settings - if for some reason the workflow is missing or haven't been run, the package will need to be manually set as public in github actions
```

Comment:
- should create a package ?
- or hasn't been run ?
- set as public (with link:) https://github.com/orgs/DFE-Digital/packages